rskj-core/src/test/java/co/rsk/core/SnapshotManagerTest.java

```diff
@@ -133,7 +140,6 @@ public void revertToNonExistentSnapshot() {
     @Test
     public void revertToSnapshot() {
         addBlocks(10);
-
```

Comment:
More tests besides cosmetic changes would be nice :) also, documenting why some tests are there (specially the surprising ones found on this PR) would be great.

Reply:
Cosmetic changes were due to the IDE auto-formatting, the only change was on line 379: `Assert.assertFalse(processor.isReadyToAcceptBlocks());`

This PR is preliminar to showcase the spotted scenarios, not production ready, so changes are expected and also more tests.